packages/search-ui/src/__tests__/SearchDriver.test.ts

```diff
@@ -334,9 +334,9 @@ describe("searchQuery config", () => {
       ]);
     });
 
-    it("will remove filters parameter from queryConfig since its merged into state", () => {
+    it("will not remove filters parameter from queryConfig", () => {
```

Comment:
this test was changed as the queryConfig.filters still remain as its needed by the connectors